presto-product-tests/README.md

```diff
@@ -1,125 +1,106 @@
 # Presto product tests
 
-## Test configuration
+## Installing dependencies
 
-Test assume that you have hadoop and Presto cluster running. 
+*Running product test requires at least 4GB free memory to be available.*
 
-To make it work you need either:
- - define environment variables for hadoop ${HADOOP_MASTER} and Presto ${PRESTO_MASTER} with their IP numbers
- - create test-configuration-local.yaml with following (example content):
+#### On GNU/Linux
+* [```docker >= 1.10```](https://docs.docker.com/installation/#installation)
 
-```
-databases:
-  hive:
-    host: 172.16.2.10
-
-  presto:
-    host: 192.168.205.1
-```
-
-## Running tests
+    ```
+    wget -qO- https://get.docker.com/ | sh
+    ```
 
-Product tests are not run by default. To start them use run following command:
+* [```docker-compose >= 1.60```](https://docs.docker.com/compose/install/)
 
-```
-java -jar target/presto-product-tests-*-executable.jar --config-local tempto-configuration-local.yaml
-```
+    ```
+    pip install docker-compose
+    ```
 
-## Running tests with Using preconfigured docker based clusters
+#### On Mac OS X
 
-In case you do not have an access to hadoop cluster or do not want to spent your time on configuration, this section
-describes how to run product tests with preconfigured docker based clusters.
+* [```docker-toolbox >= 1.10```](https://www.docker.com/products/docker-toolbox)
 
-### Prerequisites
+On Mac OS X installing docker-toolbox gives access to preconfigured bash environment
+with ```docker``` and ```docker-compose``` available. To start this bash environment
+select "Docker Quickstart Terminal" from Launchpad. Note that all commands given in
+further parts of this docs should be run from this environment.
 
-* docker >= 1.10
+##### Setting up virtual machine for docker
 
-[https://docs.docker.com/installation/#installation](https://docs.docker.com/installation/#installation)
+The default ```docker-toolbox``` setting should be decent in most cases, but setting
+up docker virtual machine to have at least 4GB is required.
 
-For linux users:
-```
-wget -qO- https://get.docker.com/ | sh
-```
+* To create virtual machine with name <machine> (required one time):
```

Comment:
Btw. I think we can do one of two things:
 1. Assume that user knows Docker and use terms that Docker user should know - then the documentation may be complete and precise but requires additional knowledge. This would be better for someone who want to know how it works and possibly develop it.
 2. Assume that user does not know Docker and give him commands that will allow him to use product tests as basic setup that will just work. That way it will be easy to use product-test but the documentation cannot be complete to keep it simple. This would be better for someone who just want to use it.

We've chosen (2), so there are some understatements in docs. However if you want to "playing dumb here." you should just copy command and see if it works - we cannot explain every command in details and keep documentation short and simple at the same time.

Maybe we can create some other documentation that will be about how it works instead of how to use it - but it's not this one I think.